packages/backend/src/services/change-history.service.ts

```diff
@@ -47,6 +51,54 @@ export class ChangeHistoryService {
     });
   }
 
+  async rollBackToCommit(gitHash: string, user: User, insight: Insight): Promise<void> {
+    const gitUrl = insight.repository.cloneUrl;
+    const { githubPersonalAccessToken } = user;
+    let insightYaml;
+    try {
+      insightYaml = await GitInstance.rollBackCommit({ gitHash, gitUrl, user });
+
+      this.activityService.recordActivity(ActivityType.EDIT_INSIGHT, user, {
+        insightId: insight.insightId,
+        insightName: insight.name
+      });
```

Comment:
Recording an `EDIT_INSIGHT` activity after a successful roll back. 
Although I just noticed we don't do that when we update an Insight.